spec/spec_helper.rb

```diff
@@ -14,16 +13,17 @@ module SpecHelper
 end
 
 spec_helper_glob = File.expand_path('{support,shared}/*.rb', __dir__)
-Dir.glob(spec_helper_glob).map(&method(:require))
+Dir.glob(spec_helper_glob).sort.each(&method(:require))
```

Comment:
`Lint/NonDeterministicRequireOrder` complains about `each`, but didn’t notice the `map`…

Reply:
I guess our `RSpec/IteratedExpectation` would also miss `map`.
But we have some code in [`RSpec/EmptyExampleGroup`](https://github.com/rubocop-hq/rubocop-rspec/blob/c3fa5ba4ccd5b3cffd58c55ca7fdde63614f2501/lib/rubocop/cop/rspec/empty_example_group.rb#L93) that is sophisticated enough to detect different iterators]((https://github.com/rubocop-hq/rubocop-rspec/blob/c3fa5ba4ccd5b3cffd58c55ca7fdde63614f2501/spec/rubocop/cop/rspec/empty_example_group_spec.rb#L171).